edition = "2023";

message SampleArgs {
  string some_string = 1;
  int64 some_int = 2;
}
